src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -94,17 +126,51 @@ def update_data(self):
             else:
                 continue
 
-            vlanmac = self.fdb_vlanmac(fdb)
+            vlanmac, vlanid = self.fdb_vlanmac(fdb)
             if not vlanmac:
                 mibs.logger.error("SyncD 'ASIC_DB' includes invalid FDB_ENTRY '{}': failed in fdb_vlanmac().".format(fdb_str))
                 continue
+            self.tp_fdb_status_map[vlanmac] = self.get_tp_fdb_status(ent, fdb)
             self.vlanmac_ifindex_map[vlanmac] = port_index
             self.vlanmac_ifindex_list.append(vlanmac)
+
+            try:
+                ent_type = ent["SAI_FDB_ENTRY_ATTR_TYPE"]
+                if ent_type == 'SAI_FDB_ENTRY_TYPE_DYNAMIC':
+                    if vlanid in self.vlan_dynamic_count_map:
+                        self.vlan_dynamic_count_map[vlanid] = self.vlan_dynamic_count_map[vlanid] + 1
+                    else:
+                        self.vlan_dynamic_count_map[vlanid] = 1
+                self.vlan_id_list.append(vlanid)
+            except:
+                continue
+        self.vlan_id_list.sort()
+        self.vlan_id_list = [(i,) for i in self.vlan_id_list]
+        mibs.logger.debug('vlan_dynamic_count_map={}'.format(self.vlan_dynamic_count_map))
+
         self.vlanmac_ifindex_list.sort()
 
     def fdb_ifindex(self, sub_id):
         return self.vlanmac_ifindex_map.get(sub_id, None)
 
+    def fdb_status(self, sub_id):
+        return self.tp_fdb_status_map.get(sub_id, None)
+
+    def get_vlan_version(self):
+        return 2
```

Comment:
>2 [](http://example.com/codeflow?start=15&length=1)

Explain magic number?

Reply:
The IEEE 802.1Q VLAN Version number. Reported as “1” by Bridges that support only SST
operation, and reported as “2” by Bridges that support MST operation;